src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1364,6 +1368,24 @@ private function registerLockConfiguration(array $config, ContainerBuilder $cont
         }
     }
 
+    private function registerMessageConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
+    {
+        $loader->load('message.xml');
+
+        $messageToSenderMapping = array();
+        foreach ($config['routing'] as $message => $messageConfiguration) {
+            $messageToSenderMapping[$message] = array_map(function (string $service) {
+                if (is_string($service)) {
```

Comment:
`$service` is always a string

Reply:
No, it can be `null`.

Reply:
then `function (?string $service)`

Reply:
Oh, good catch.

Reply:
Let's be explicit then: `if (null !== $service)`.